telegram/ext/_application.py

```diff
@@ -204,6 +204,11 @@ class Application(Generic[BT, CCT, UD, CD, BD, JQ], AbstractAsyncContextManager)
         post_shutdown (:term:`coroutine function`): Optional. A callback that will be executed by
             :meth:`Application.run_polling` and :meth:`Application.run_webhook` after shutting down
             the application via :meth:`shutdown`.
+        post_stop (:term:`coroutine function`): Optional. A callback that will be executed by
+            :meth:`Application.run_polling` and :meth:`Application.run_webhook` after stopping
+            the application via :meth:`stop`.
+
+            ..versionadded:: 21.0
```

Comment:
```suggestion
            .. versionadded:: 21.0
```

Reply:
I don't think that we want to make the next release breaking again, so the next version will be 20.1 :D

Reply:
Alright!